report/report.md

```diff
@@ -0,0 +1,898 @@
+            + \ip{l(\vec{c}_i) + \xi^{-1}_{i+1} r(\vec{c}_i)}{l(\vec{z}_i) + \xi_{i+1} r(\vec{z}_i)} H'\\
+          &= \ip{l(\vec{c}_i)}{l(\vec{G}_i)} + \xi_{i+1} \ip{l(\vec{c}_i))}{r(\vec{G}_i}
+            + \xi^{-1}_{i+1} \ip{r(\vec{c}_i)}{l(\vec{G}_i)} + \ip{r(\vec{c}_i)}{r(\vec{G}_i)}\\
+          &+ (\ip{l(\vec{c}_i)}{l(\vec{z}_i)} + \xi_{i+1} \ip{l(\vec{c}_i)}{r(\vec{z}_i)} 
+            + \xi^{-1}_{i+1} \ip{r(\vec{c}_i)}{l(\vec{z}_i)} + \ip{r(\vec{c}_i)}{l(\vec{z}_i)}) H'
+\end{alignedat}
+$$
+
+If we group these terms:
+
+$$
+\begin{alignedat}[b]{4}
+  C_{i+1} &= \ip{l(\vec{c}_i)}{l(\vec{z}_i)}  &&+ \ip{r(\vec{c}_i)}{r(\vec{G}_i)}     &&+ \xi_{i+1} \ip{l(\vec{c}_i)}{r(\vec{G}_i)}    &&+ \xi^{-1}_{i+1} \ip{r(\vec{c}_i)}{l(\vec{G}_i)} \\
+          &+ (\ip{l(\vec{c}_i)}{l(\vec{z}_i)} &&+ \ip{r(\vec{c}_i)}{r(\vec{z}_i)}) H' &&+ \xi_{i+1} \ip{l(\vec{c}_i)}{r(\vec{z}_i)} H' &&+ \xi^{-1}_{i+1} \ip{r(\vec{c}_i)}{l(\vec{z}_i)} H' \\
+          &= C_i                              &&                                      &&+ \xi_{i+1} R_i                                &&+ \xi^{-1}_{i+1} L_i \\
+          &\mkern-18mu\mkern-18mu \textbf{Where:} && && && \\
+  L_i     &= \ip{r(\vec{c}_i)}{l(\vec{G}_i)} &&+ \ip{r(\vec{c}_i)}{l(\vec{z}_i)} H' && && \\
+  R_i     &= \ip{l(\vec{c}_i)}{r(\vec{G}_i)} &&+ \ip{l(\vec{c}_i)}{r(\vec{z}_i)} H' && && 
+\end{alignedat}
+$$
+
+We see why $\vec{L}, \vec{R}$ is defined the way they are. They help
+the verifier check that the original relation hold, by showing it for the
+compressed form $C_{i+1}$. $\vec{L}, \vec{R}$ is just the minimal information
+needed to communicate this fact.
+
+This leaves us with the following vectors (notice the slight difference in length):
+
+$$
+\begin{alignedat}[b]{1}
+  \vec{L}    &= (L_1, \dots, L_{\lg(n)}) \\
+  \vec{R}    &= (R_1, \dots, R_{\lg(n)}) \\
+  \vec{C}    &= (C_0, \dots, C_{\lg(n)}) \\
+  \vec{\xi}  &= (\xi_0, \dots, \xi_{\lg(n)}) \\
+\end{alignedat}
+$$
+
+This means an honest prover will indeed produce $\vec{L}, \vec{R}$ s.t. $C_{\lg(n)} = C_0 + \sum^{\lg(n)-1}_{i=0} \xi^{-1}_{i+1} L_i + \xi_{i+1} R_i$
+
+Let's finally look at the left-hand side of the verifying check:
+
+\begin{align*}
+  C_{\lg(n)} &= C_0 + \sum^{\lg(n)-1}_{i=0} \xi^{-1}_{i+1} L_i + \xi_{i+1} R_i && \\
+             \intertext{The original definition of $C_i$:}
+             &= \ip{\vec{c}_{\lg(n)}}{\vec{G}_{\lg(n)}} + \ip{\vec{c}_{\lg(n)}}{\vec{z}_{\lg(n)}} H' \\
+             \intertext{Vectors have length one, so we use the single elements $c^{(0)}, G^{(0)}, c^{(0)}, z^{(0)}$ of the vectors:}
+             &= c^{(0)}G^{(0)} + c^{(0)}z^{(0)} H'                                                   \\
+             \intertext{The verifier has $c^{(0)} = c, G^{(0)} = U$ from $\pi \in \EvalProof$:}
+             &= cU + cz^{(0)} H'                                                                     \\
+             \intertext{Then, by construction of $h(X) \in \Fb^d_q[X]$}
```

Comment:
Not sure on this, "by construction" seems incomplete, but I have a hard time proving/formalizing why this is.